vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -232,6 +232,26 @@ function Get-PrivatePackagesToCacheOnVHD {
     }
 }
 
+function Get-ToolsToVHD {
+    $cacheDir = "c:\toolscache"
+    if (!(Test-Path -Path $cacheDir)) {
+        New-Item -ItemType Directory -Path $cacheDir | Out-Null
+    }
+
+    $toolsDir = "c:\tools" # discuss
```

Comment:
could we use aks as prefix? NOTE: Remove the comment.

Reply:
Updated